DigitalExcellence/dex-frontend
# Feature/213 button validation

## Description
I added validation so that buttons that edit / delete / highlight / embed the project will only be shown when you are allowed.

## Type of change


## Checklist

-   [x ] My code follows the style guidelines of this project

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

go to a project detail page.
you should not see the buttons.
then login as normal user, still no buttons
then create a project, now you should see edit delete and embed.
then login as administrator or pr and you should also see the highlight buttons.

## Link to issue

Closes: #213
